setup.py

```diff
@@ -288,8 +288,15 @@ def run(self):
     'numpydoc',
 ]
 
+extra_require = {
+    'test:python_version == "2.7"': ['mock'],
```

Comment:
How does this syntax work? I'm just curious and I'd like to understand how it works :-)